sopel/modules/help.py

```diff
@@ -13,16 +13,30 @@
 import textwrap
 import collections
 import requests
+import socket
 
 from sopel.logger import get_logger
 from sopel.module import commands, rule, example, priority
+from sopel.config.types import (
+    StaticSection, ChoiceAttribute
+)
+
+
+class HelpSection(StaticSection):
+    output = ChoiceAttribute('output', ["ptpb", "0x0", "hastebin", "termbin"], default='ptpb')
+
+
+def configure(config):
+    config.define_section('help', HelpSection)
+
```

Comment:
Could be nice if there was a prompt to select the provide in the config wizard :smile: 
Maybe like so:

```
config.help.configure_setting(
    'output',
    'Pick a pastebin provider: ptpb, 0x0, hastebin, termbin: '
)
```
? You can likely use format string to put the choices in the prompt.